tst/EnergyPlus/unit/WaterThermalTanks.unit.cc

```diff
@@ -496,3 +500,265 @@ TEST_F( HVACFixture, HPWHWrappedDummyNodeConfig )
+		"    0,                       !- Crankcase Heater Capacity {W}",
+		"    0,                       !- Maximum Ambient Temperature for Crankcase Heater Operation {C}",
+		"    WetBulbTemperature,      !- Evaporator Air Temperature Type for Curve Objects",
+		"    HPWH-Cap-fT,             !- Heating Capacity Function of Temperature Curve Name",
+		"    ,                        !- Heating Capacity Function of Air Flow Fraction Curve Name",
+		"    HPWH-COP-fT;             !- Heating COP Function of Temperature Curve Name",
+		"Curve:Biquadratic,",
+		"    HPWH-Cap-fT,             !- Name",
+		"    0.563,                   !- Coefficient1 Constant",
+		"    0.0437,                  !- Coefficient2 x",
+		"    0.000039,                !- Coefficient3 x**2",
+		"    0.0055,                  !- Coefficient4 y",
+		"    -0.000148,               !- Coefficient5 y**2",
+		"    -0.000145,               !- Coefficient6 x*y",
+		"    0,                       !- Minimum Value of x",
+		"    100,                     !- Maximum Value of x",
+		"    0,                       !- Minimum Value of y",
+		"    100;                     !- Maximum Value of y",
+		"Curve:Biquadratic,",
+		"    HPWH-COP-fT,             !- Name",
+		"    1.1332,                  !- Coefficient1 Constant",
+		"    0.063,                   !- Coefficient2 x",
+		"    -0.0000979,              !- Coefficient3 x**2",
+		"    -0.00972,                !- Coefficient4 y",
+		"    -0.0000214,              !- Coefficient5 y**2",
+		"    -0.000686,               !- Coefficient6 x*y",
+		"    0,                       !- Minimum Value of x",
+		"    100,                     !- Maximum Value of x",
+		"    0,                       !- Minimum Value of y",
+		"    100;                     !- Maximum Value of y",
+		"Fan:OnOff,",
+		"    HPWH Fan_1,              !- Name",
+		"    ,                !- Availability Schedule Name",
+		"    0.2349521887445888,      !- Fan Total Efficiency",
+		"    23,                      !- Pressure Rise {Pa}",
+		"    0.08542248664,           !- Maximum Flow Rate {m3/s}",
+		"    1,                       !- Motor Efficiency",
+		"    1,                       !- Motor In Airstream Fraction",
+		"    HPWH CoilAirOutlet FanAirInlet_1,  !- Air Inlet Node Name",
+		"    HPWH Air Outlet Node_1,  !- Air Outlet Node Name",
+		"    ,                        !- Fan Power Ratio Function of Speed Ratio Curve Name",
+		"    ,                        !- Fan Efficiency Ratio Function of Speed Ratio Curve Name",
+		"    Water Heating_1;         !- End-Use Subcategory",
+	});
+
+	ASSERT_FALSE( process_idf( idf_objects ) );
+
+	ASSERT_FALSE( WaterThermalTanks::GetWaterThermalTankInput() );
+
+	WaterThermalTanks::WaterThermalTankData & Tank = WaterThermalTank( 1 );
+	WaterThermalTanks::HeatPumpWaterHeaterData & HPWH = HPWaterHeater( Tank.HeatPumpNum );
+	DXCoils::DXCoilData & Coil = DXCoils::DXCoil( HPWH.DXCoilNum );
```

Comment:
It's used [here](https://github.com/NREL/EnergyPlus/blob/ee204a1fadb6d5cad968e31e8300c32556c98b94/tst/EnergyPlus/unit/WaterThermalTanks.unit.cc#L745).